articles/understanding-differential-privacy/index.md

```diff
@@ -0,0 +1,297 @@
+<!--more-->
+
+### Table of contents
+
+- [Introduction](#introduction)
+- [Differential Privacy](#differential-privacy)
+- [Differencing attacks](#differencing-attacks)
+- [Implementing differential privacy](#implementing-differential-privacy)
+- [Types of Differential privacy](#types-of-differential-privacy)
+- [Conclusion](#conclusion)
+- [Further reading](#further-reading)
+
+### Introduction
+
+In this digital era, most of the companies are data-driven. Our data is harvested to keep growing their economic incentives, in exchange for services provided to us. Research institutions often use and share data containing confidential information about individuals. Improper disclosure of such data can have adverse consequences for a data subject’s private information, or even lead to civil liability or bodily harm.
+
+In the late 2000s, Netflix ran a competition for building a recommendation system. To build, they released a large dataset by anonymizing the data which didn’t leak the movie names or users' ratings. Two professors from the University of Texas [de-anonymized](https://www.cs.utexas.edu/~shmat/shmat_oak08netflix.pdf) them completely by comparing them with IMDB ratings, thus revealing the personal information about the dataset. Data privacy is compromised! Similarly, one such data breach of health records happened in 1997 too.
+
+This sort of data leakage is very alarming both to the users as well as organizations. To resolve this, we can either completely randomize the way the data is sent, using keys that are only decipherable at either end (Differential Learning). Well, there is an alternative to use Federated learning, where we bring the model to the data instead of vice-versa. Fundamentally, we must understand the concept of Differential Privacy to check for leakage in data.
+
+### Differential Privacy
+
+#### Former definition
+
+*Anything that can be learned from a statistical database can be learned without accessing the database*
+
+This definition does not hold always. We need to consider two things, first is people and the second is data. We need to protect people's rights as well as predict rightly using their data. In certain cases, the whole database must be kept private, to protect the information, because there are chances of data leaks.
+
+#### Modern definition
+
+*It’s a promise made by a data holder to a data subject, that “You will not be affected, adversely or otherwise, by allowing your data to be used for study analysis, no matter what other studies, datasets or information is available”
+-Cynthia Dwork (Godfather of Differential Privacy)*
+
+A simple way to explain the modern definition is
```

Comment:
A simple way to explain the modern definition is -> A simple way to explain the modern definition is: